universal_transfer_operator/src/universal_transfer_operator/__init__.py

```diff
@@ -0,0 +1,20 @@
+"""A decorator that allows users to run SQL queries natively in Airflow."""
```

Comment:
Not true anymore for UTO

Reply:
[Done](https://github.com/astronomer/astro-sdk/pull/1492/commits/c2e08eaec89c463d8e83e7a8fa523231e56ede4f)